docs/sources/administration/configuration.md

```diff
@@ -1115,6 +1115,24 @@ For more information about the Grafana 8 alerts, refer to [Unified Alerting]({{<
 
 Specify the frequency of polling for admin config changes. The default value is `60`.
 
+### execute_alerts
+
+Turns off alert rule execution, but Grafana 8 Alerts are still visible in the Grafana UI. Default is `true`. If it's true (the default) then the [respective legacy option]({{< relref "#execute_alerts-1">}}) is applied.
+
+### evaluation_timeout_seconds
+
+Sets the alert calculation timeout. Default value is `30`. If it's `30` (the default) then the [respective legacy option]({{< relref "#evaluation_timeout_seconds-1">}}) is applied.
```

Comment:
I forget if the dataproxy timeout applies here? If so may we should cross reference it since the queries will timeout even if this is higher (if that is true)

Reply:
good point! I'll try to set a low proxy timeout to try it.